data/en/spreadsheetgetcellformula.json

```diff
@@ -25,6 +25,13 @@
 			"result":"",
 			"runnable":true
+		},
+		{
+			"title": "Example for getting cell formula",
+			"description": "Example of getting Excel spreadsheet cell value and formula",
+			"code": "<cfscript>\ncolumnsList = \"Name,Employed,Sales,Earnings,Some Value\";\nemployees = [\n    { \"Name\": \"Robert\",\"Employed\":\"Y\",\"Sales\":1,\"Earnings\":500 },\n    { \"Name\": \"Wilma\",\"Employed\":\"Y\",\"Sales\":15,\"Earnings\":5500 },\n    { \"Name\": \"Eric\",\"Employed\":\"Y\",\"Sales\":3,\"Earnings\":1200 },\n    { \"Name\": \"Fred\",\"Employed\":\"Y\",\"Sales\":1,\"Earnings\":500 },\n    { \"Name\": \"Dillan\",\"Employed\":\"Y\",\"Sales\":15,\"Earnings\":5500 },\n    { \"Name\": \"Dana\",\"Employed\":\"Y\",\"Sales\":3,\"Earnings\":1200 },\n    { \"Name\": \"Sandy\",\"Employed\":\"Y\",\"Sales\":1,\"Earnings\":500 },\n    { \"Name\": \"Ellen\",\"Employed\":\"Y\",\"Sales\":15,\"Earnings\":5500 },\n    { \"Name\": \"Richard\",\"Employed\":\"Y\",\"Sales\":3,\"Earnings\":1200 }\n];\n\n// create a spreadsheet\ns = SpreadsheetNew(\"Example\");\n// add the header row\nspreadsheetAddRow(s, columnsList);\n// Add some employee data\nemployees.each(function(record,row){\n    spreadsheetAddRow(s, \"\", row + 1);\n    // Add the row data by column name\n    ListToArray(columnsList).each(function(columnName,col){\n        if(structKeyExists(record,columnName)){\n            SpreadsheetSetCellValue(s, record[columnName], row + 1, col);\n        }\n    });\n});\n// Capture the number of rows plus the header row\nnumRows = ArrayLen(employees) + 1;\n\n// Create some random data in the 5th column\nfor (i=2; i<= numRows; i=i+1)\n    SpreadsheetSetCellValue(s,i,i,5);\n// add a totals row\nspreadsheetAddRow(s, \"\", numRows + 1);\n\n// Set the formula for the cell in the last row, column 3 to be the sum of\n// 'Sales'\nSpreadsheetSetCellFormula(s,\"SUM(#Chr(64+3)#2:#Chr(64+3)#10)\",numRows+1,3);\n// Set the formula for the cell in the last row, column 4 to be the sum of\n// 'Earnings'\nSpreadsheetSetCellFormula(s,\"SUM(#Chr(64+4)#2:#Chr(64+4)#10)\",numRows+1,4);\n// Set the formula for the cell in the last row, column 5 to be the sum of\n// 'Some Value'\nSpreadsheetSetCellFormula(s,\"SUM(#Chr(64+5)#2:#Chr(64+5)#10)\",numRows+1,5);\n\nfor (i=3; i<= 5; i=i+1){\n    //Get the formula from last row column i ex: ($E11)\n    formulaValue=SpreadsheetGetCellFormula(s,numRows+1,i);\n    //Get the value from last row column i ex: ($E11)\n    cellValue=SpreadsheetGetCellValue(s,numRows+1,i);\n    writeOutput(\"Total: \" & cellValue & \"<br>\");\n    writeOutput(\"formula: \" & formulaValue & \"<hr>\" );\n}\nwritedump(var=s,label=\"Example\");\n</cfscript>",
+			"result": "Total: 57 formula: SUM(C2:C10) Total: 21600 formula: SUM(D2:D10) Total: 54 formula: SUM(E2:E10)",
+			"runnable": "1"
```

Comment:
This should be `"runnable": true` not `"runnable": "1"`.

Reply:
Yes, this was from the previous example that was submitted.   I just saw this as well, thank you!

Reply:
Oh, it was my generated result.  Thank you for pointing this out!

Reply:
Good catch @KamasamaK, I always appreciate your input!

@carlhuth - I think you must have used the new _Add Example_ button which caused this bug in the generated json, it has been fixed already, so next time you do this you would not have the same problem.